Makefile

```diff
@@ -2,65 +2,66 @@ go_bin := "$$(go env GOPATH)/bin"
 gen_authopt_path := "$(go_bin)/protoc-gen-authoption"
 gen_zitadel_path := "$(go_bin)/protoc-gen-zitadel"
 
-now := $(shell date '+%Y-%m-%dT%T%z' | sed -E 's/.([0-9]{2})([0-9]{2})$$/-\1:\2/')
+now := $$(date '+%Y-%m-%dT%T%z' | sed -E 's/.([0-9]{2})([0-9]{2})$$/-\1:\2/')
 VERSION ?= development-$(now)
-COMMIT_SHA ?= $(shell git rev-parse HEAD)
+COMMIT_SHA ?= $$(git rev-parse HEAD)
```

Comment:
I think this can be a problem on other systems then linux

Reply:
I revert it.